sample-code/python/test/test_ios_selectors.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import os
+
+from appium import webdriver
+from helpers import take_screenhot_and_syslog
+
+
+class TestIOSBasicInteractions():
+    APP_PATH = 'http://appium.github.io/appium/assets/TestApp7.1.app.zip' if os.getenv(
```

Comment:
These variables are being repeated as well as some common capabilities. How about moving them to helpers?

Reply:
I thought so, but I also thought it will be helpful for new users to collect variables like capabilities into one file test file.
Let me consider further. (will back to here after tweaking other points)